app/upgrades/v28/upgrades_test.go

```diff
@@ -40,14 +43,46 @@ func TestKeeperTestSuite(t *testing.T) {
 func (s *UpgradeTestSuite) TestUpgrade() {
 	upgradeHeight := int64(4)
 
-	// Set state before upgrade
+	// ------- Set state before upgrade -------
 	checkRedemptionRates := s.SetupTestUpdateRedemptionRateBounds()
+	checkLockedTokens := s.SetupTestDeliverLockedTokens()
 
-	// Run upgrade
+	// ------- Run upgrade -------
 	s.ConfirmUpgradeSucceededs(v28.UpgradeName, upgradeHeight)
 
-	// Confirm state after upgrade
+	// ------- Confirm state after upgrade -------
 	checkRedemptionRates()
+	checkLockedTokens()
+}
+
+func (s *UpgradeTestSuite) SetupTestDeliverLockedTokens() func() {
+
+	// Init BaseAccount (which is the type of the account pre-upgrade)
+	deliveryAccountAddress, err := sdk.AccAddressFromBech32(v28.DeliveryAccount)
+	s.Require().NoError(err)
+	deliveryAccount := s.App.AccountKeeper.GetAccount(s.Ctx, sdk.MustAccAddressFromBech32(v28.DeliveryAccount))
```

Comment:
```suggestion
	deliveryAccount := authtypes.NewBaseAccount(deliveryAccountAddress, nil, 476582, 0)
```
I think we need to create this account, as it doesn't exist yet in the chain state

Note: we'll also have to import ```authtypes "github.com/cosmos/cosmos-sdk/x/auth/types"``` at the top of the file

Reply:
>I think we need to create this account, as it doesn't exist yet in the chain state

Are you sure? The test does run